security/security-spring/pom.xml

```diff
@@ -20,6 +20,11 @@
         <artifactId>persistence-mybatis</artifactId>
```

Comment:
We can remove this dependency as web already depends on this.

Reply:
Ok.

Reply:
Done!